src/views/payment-sheet-views/card-view.js

```diff
@@ -396,7 +396,11 @@ CardView.prototype.tokenize = function () {
     });
   }).catch(function (err) {
     self._isTokenizing = false;
-    self.model.reportError(err);
+    if (err.code === 'HOSTED_FIELDS_TOKENIZATION_FAIL_ON_DUPLICATE') {
+      self.model.reportError('hostedFieldsDuplicateCardError');
```

Comment:
If you change the translation key to be `hostedFieldsTokenizationFailOnDuplicateError`, this will work automagically without this if block.

See https://github.com/braintree/braintree-web-drop-in/blob/63cef1448628aa46c9dc5d916b83a7f45f00b93f/src/views/main-view.js#L255-L256